app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollViewModel.java

```diff
@@ -0,0 +1,48 @@
+package ch.heigvd.pro.b04.android.Poll;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+import ch.heigvd.pro.b04.android.Poll.Question.Question;
+
+
+public class PollViewModel extends ViewModel {
+    private MutableLiveData<List<Question>> queue = new MutableLiveData<>();
+
+    private MutableLiveData<Set<Question>> answeredQuestion = new MutableLiveData<>();
```

Comment:
There's no default value to the `MutableLiveData` objects, and also no way to "populate them".